packages/react-native/ios/BugsnagReactNative/BugsnagEventDeserializer.m

```diff
@@ -85,9 +85,12 @@ - (BugsnagEvent *)deserializeEvent:(NSDictionary *)payload {
                                                     threads:[self deserializeThreads:payload[@"threads"]]
                                                     session:session];
     event.context = payload[@"context"];
-    event.apiKey = payload[@"apiKey"];
     event.groupingHash = payload[@"groupingHash"];
 
+    if (payload[@"apiKey"]) {
```

Comment:
Does this get coerced to a boolen for the if statement? just wondering why it doesn't need `!= nil` check like some other fields in this method.

Reply:
yes - if the key doesn't exist, the statement resolves to nil, which is 'falsy'. Just about any other value is truthy.

Reply:
On that basis I'm happy to approve this myself.